subsys/power/device.c

```diff
@@ -32,6 +32,17 @@ static const char core_devices[NUM_CORE_DEVICES][MAX_DEV_NAME_LEN] = {
 	"sys_clock",
 	"UART_0",
 };
+#elif defined(CONFIG_SOC_SERIES_STM32L4X) \
```

Comment:
Let's stick to L4 only for now

Reply:
ok, 
#elif defined(CONFIG_SOC_SERIES_STM32L4X)